Web/CaseStudyAssignment-4/CaseStudyAssignment-4/WebContent/addImageOfCar.jsp

```diff
@@ -0,0 +1,84 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<%@page session="false"%>
+
+<%
+	HttpSession session = request.getSession();
+	session.setAttribute("make", request.getParameter("make"));
+%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+		<title>Car Portal</title>
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+		<link rel="stylesheet" type="text/css" href="css/styleLogin.css">
+	</head>
+	<body>
+		<header> 
+		<br />
+		<span id="logo">Car Portal</span> 
+		<%
+	 	if (session.getAttribute("username") != null) {
```

Comment:
ACTION: Can we create a common view for header and use that on all pages instead of duplicating same code on every page?